src/lib.rs

```diff
@@ -148,7 +148,7 @@ pub enum DeepgramError {
     #[cfg(feature = "listen")]
     /// Something went wrong with WS.
     #[error("Something went wrong with WS: {0}")]
-    WsError(#[from] TungsteniteError),
+    WsError(#[from] Box<TungsteniteError>),
```

Comment:
This is a semver breaking change for anyone trying to pattern match into this error type.  Not a big deal, but we may want to delay merging this PR until we want to make a breaking change for other reasons.  In the meantime, I'd be okay with suppressing the lint.